devicetypes/smartthings/testing/smartsense-virtual-open-closed.src/simulated-contact-sensor.src/simulated-contact-sensor.groovy

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
What is that file? It doesn't look as part of DTH for ORVIBO. Remove it from PR is not necessary

Reply:
That those were added by mistake,and I have deleted those two.